tests/Integration/Behaviour/Features/Context/CurrencyFeatureContext.php

```diff
@@ -121,4 +140,79 @@ public function checkCurrencyWithNameExists($currencyName)
     {
         $this->checkFixtureExists($this->currencies, 'Currency', $currencyName);
     }
+
+    /**
+     * @Then currency :reference should be :isoCode
+     */
+    public function assertCurrencyIsoCode($reference, $isoCode)
+    {
+        $currency = $this->domainCurrencyFeatureContext->getCurrencyFromRegistry($reference);
+
+        if ($currency->iso_code !== $isoCode) {
+            throw new RuntimeException(sprintf(
+                'Currency "%s" has "%s" iso code, but "%s" was expected.',
+                $reference,
+                $currency->iso_code,
+                $isoCode
+            ));
+        }
+    }
+
+    /**
+     * @Then currency :reference should have status :status
```

Comment:
(enabled|disabled) so you dont need the check `if (!in_array($status, $statuses)) {` and writing steps will enable autocompletion

Reply:
Sure, but why it's better to use `"(.*)" ` instead of `:reference`?

Reply:
I don't think colons can be used in regexp format, since it is specific to gherkins/behat